### PR TITLE
[FEATURE] Library support

### LINKED ISSUE #22: Support for libraries?
Any plans to add library support? For example, to include the 'drawing' library, the script tag would be:

    <script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?libraries=drawing"></script>

### PR DESCRIPTION
**Pull request by sbsurf**
Put this together very quickly. Hopefully this will do the trick.

This fixes #22.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,6 +10,9 @@ module.exports = {
       if (google.key) {
         params.push('key=' + encodeURIComponent(google.key));
       }
+      if (typeof google.libraries !== 'undefined' && google.libraries.length > 0) {
+        params.push('libraries=' + google.libraries.join());
```
Comment (huafu) [inline index.js]: I have a branch where I have this too, to handle geolocation. Anyway, you need to join with `','` and also to escape it.

**Comment by huafu** [comment]
Forget it, I need to handle #24 so I'll handle #22 as well, thanks!

File: index.js (hunk 1 above)
Comment (sbsurf) [inline index.js]: Actually, calling **join()** puts in the commas by default.

Escaping is a good idea, though.

Comment (huafu) [inline index.js]: wow! Haha I never used it without parameter, and thought that it'd join with an empty string! Comma is a weird default from a developer point of view (or maybe just mine lol), sounds more like a literate user default lol

**Pull request closed without merging**